server/src/main/java/org/elasticsearch/index/IndexingSlowLog.java

```diff
@@ -90,7 +90,11 @@ public final class IndexingSlowLog implements IndexingOperationListener {
         Property.IndexSettingDeprecatedInV7AndRemovedInV8
     );
 
-    private final Logger indexLogger;
+    private static final Logger indexLogger = LogManager.getLogger(INDEX_INDEXING_SLOWLOG_PREFIX + ".index");
+    static {
+        Loggers.setLevel(indexLogger, Level.TRACE);
```

Comment:
I see it was already this way, but why are we initializing to trace level? That seems rather verbose...

Reply:
I must admit I have no idea. My best guess would be: these were per-index instances before and we manually configured them for that reason instead of putting this in the log config?